cinema/serializers.py

```diff
@@ -112,14 +112,17 @@ class Meta:
 
 class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
-        data = super(TicketSerializer, self).validate(attrs=attrs)
-        Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
-            ValidationError
+        ticket = Ticket(
```

Comment:
This instantiation of a Ticket object inside the validate method seems unnecessary. You could directly use the `attrs` dictionary to validate the fields.